TweetBook/Data/DataContext.cs

```diff
@@ -11,5 +11,16 @@ public DataContext(DbContextOptions<DataContext> options) : base(options)
         }
         public DbSet<Post> Posts { get; set; }
         public DbSet<RefreshToken> RefreshTokens { get; set; }
+        public DbSet<Tag> Tags { get; set; }
+        public DbSet<PostTag> PostTags { get; set; }
+
+        protected override void OnModelCreating(ModelBuilder builder)
+        {
+            base.OnModelCreating(builder);
+
+            builder.Entity<PostTag>()
+                .Ignore(xx => xx.Post)
+                .HasKey(x => new { x.PostId, x.TagName });
```

Comment:
Here this x is a PostTag, but x is not a very explicit name for PostTag, is it? 
Please try to use descriptive variables everywhere.